### ISSUE TITLE
rspec rest-spec Error!

### ISSUE THREAD

**Issue by abol3z** [bemyeyes/bemyeyes-server#18]
hello,
I am trying to run bemyeyes server on vagrant, 
when i'm trying to run the `rspec test-spec` command, i get this message

![screenshot 2015-02-23 13 59 27](https://cloud.githubusercontent.com/assets/3227740/6326655/feccfea2-bb64-11e4-84f7-61667ed38824.png)

what could I be missing??
another thing, how can I make sure that the server is running and accessible from the iphone app?

**Comment by khebbie** [comment]
At the moment you need to install growl notify to have it running.
To give acces to the iphone app, you could use a tool like ngrok or one of the many tools alike

**Comment by abol3z** [comment]
thanks for the response, but I didn't understand you well.
I'm running the server now on windows pc, I installed vagrant and I run the server on it.
now should I install these tools you said about on my windows pc? or on the server?
sorry for that but I'm new to the server things!

**Comment by abol3z** [comment]
Thanks for the response, I took a look at these tools but I didn't understand how to use them to solve my problem.
what I'm doing is trying to run the server on my laptop in vagrant machine as you mentioned in the README, and then connect to it from the iphone app locally throw my LAN network. 
I did everything in the README until the testing section, when I run the first command `rspec` I'm getting a result, but when I run the second command `rspec rest-spec` i'm getting the error in the image above.
I am able to ping the vagrant machine that runs the `bemyeyes-server` from other devices in my LAN network, so I think it is accessible.
I downloaded the iphone-app project, I changed the URL to the vagrant machine ip address, and I run the app, then when I try to register i'm getting `cannot connect to the server` error.
Is there something that I should do to be able to run the system locally and it is not mentioned the README file?
